src/boolean_result.cpp

```diff
@@ -253,7 +253,7 @@ void AddNewEdgeVerts(
 #if (MANIFOLD_PAR == 1) && __has_include(<tbb/tbb.h>)
   // parallelize operations, requires concurrent_map so we can only enable this
   // with tbb
-  if (!ManifoldParams().deterministic && p1q2.size() > kParallelThreshold) {
+  if (p1q2.size() > kParallelThreshold) {
```

Comment:
@pca006132 Actually, switching this to serial had the smallest performance impact of anything, but for now I'm going with the `determinism == false` path for everything.